site/src/comparison.rs

```diff
@@ -523,8 +523,21 @@ pub fn write_summary_table(
     primary: &ArtifactComparisonSummary,
     secondary: &ArtifactComparisonSummary,
     with_footnotes: bool,
+    include_metric: bool,
     result: &mut String,
 ) {
+    let metric = include_metric
+        .then(|| {
+            primary
+                .relevant_comparisons
+                .first()
+                .or(secondary.relevant_comparisons.first())
+                .map(|m| m.metric.as_str())
+        })
+        .flatten()
+        // we want at least 10 spaces to accommodate "count[^2]"
```

Comment:
I now realized that this comment is wrong, `count[^2]` is no longer a row header, but a column header. But it's not crucial to change it now.